frontend-web/webclient/app/Applications/Jobs/View.tsx

```diff
@@ -319,9 +320,15 @@ export function View(props: {id?: string; embedded?: boolean;}): React.ReactNode
     const location = useLocation();
     const appNameHint = getQueryParamOrElse(location.search, "app", "");
     const [jobFetcher, fetchJob] = useCloudAPI<Job | undefined>({noop: true}, undefined);
+    const [, invokeCommand] = useCloudCommand();
     const job = jobFetcher.data;
+    const [interfaceTargets, setInterfaceTargets] = useState<InterfaceTarget[]>([]);
 
-    const interfaceLinks = useRedirectToWebLinks(job);
+    useEffect(() => {
+        if (!job) return;
+
+        findInterfaceTargets(job, invokeCommand).then(setInterfaceTargets);
```

Comment:
I haven't checked, but it wouldn't surprise me if this function is never reinvoked when new updates arrive over the websocket connection. This function might need to live closer to the other update parsing code.